teep.cddl

```diff
@@ -36,33 +36,35 @@ TEEP-TYPE-teep-error = 5
 TEEP-TYPE-teep-success = 6
 
 version = uint .size 4
-ext-info = uint
+ext-info = uint 
```

Comment:
Martin's I-D template toolchain might complain about trailing spaces at some point. But there is make fix-lint